### ISSUE TITLE
Adding own dynamic wallpapers

### ISSUE THREAD

**Issue by Mr3Sepz** [manishprivet/dynamic-gnome-wallpapers#7]
I created a dynamic wallpaper for gnome that is not listed here. 
Is there any way I could/should share it, so that people can use it?

**Comment by manishprivet** [comment]
Yes sure @Mr3Sepz, Can you share the link here and I will add, or you can create a PR

**Comment by Mr3Sepz** [comment]
I made the xml file already myself. The wallpaper does not exist on dynamicwallpaperclub or anything. 

How do I make this acessible for others? 
Should I modify the xml file or just the send you the individual pictures or something?

**Comment by manishprivet** [comment]
@Mr3Sepz You can send the zip file to me over mail me@manishk.dev
